<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plotter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/chart_curve</normaloff>:/chart_curve</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>2</number>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Plots</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PlotManagerClass" name="plotsManager">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>400</height>
           </size>
          </property>
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="uniformRowHeights">
           <bool>true</bool>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionData_load"/>
    <addaction name="actionData_save"/>
    <addaction name="actionSetup"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionAdd_Asset"/>
    <addaction name="actionAdd_Bus"/>
    <addaction name="actionChange"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_Plot"/>
    <addaction name="actionRemove"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionPlots_to_Grid"/>
    <addaction name="actionClose_Plots"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Plot"/>
   <addaction name="actionRemove"/>
   <addaction name="actionPlots_to_Grid"/>
   <addaction name="actionClose_Plots"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionData_load"/>
   <addaction name="actionData_save"/>
   <addaction name="actionSetup"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Asset"/>
   <addaction name="actionAdd_Bus"/>
   <addaction name="actionChange"/>
  </widget>
  <action name="actionAdd_Plot">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/chart-curve-add-icon.png</normaloff>:/icons/chart-curve-add-icon.png</iconset>
   </property>
   <property name="text">
    <string>Add &amp;Plot</string>
   </property>
   <property name="toolTip">
    <string>Adding new plot</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/remove.png</normaloff>:/icons/remove.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Remove</string>
   </property>
   <property name="toolTip">
    <string>Removing selected</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Info.png</normaloff>:/icons/Info.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAdd_Asset">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/pitons.png</normaloff>:/icons/pitons.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add Pipe Asset</string>
   </property>
   <property name="toolTip">
    <string>Adding Serpent pipe asset</string>
   </property>
  </action>
  <action name="actionData_save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Save_16x16.png</normaloff>:/icons/Save_16x16.png</iconset>
   </property>
   <property name="text">
    <string>D&amp;ata save</string>
   </property>
   <property name="toolTip">
    <string>Save data to disk</string>
   </property>
  </action>
  <action name="actionData_load">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Get_16x16.png</normaloff>:/icons/Get_16x16.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Data load</string>
   </property>
   <property name="toolTip">
    <string>Load data from disk</string>
   </property>
  </action>
  <action name="actionClose_Plots">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/hide plots.png</normaloff>:/icons/hide plots.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close Plots</string>
   </property>
   <property name="toolTip">
    <string>Closing all plots</string>
   </property>
  </action>
  <action name="actionAdd_Bus">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/bus.png</normaloff>:/icons/bus.png</iconset>
   </property>
   <property name="text">
    <string>Add &amp;BusMT Asset</string>
   </property>
   <property name="toolTip">
    <string>Adding BusMT asset</string>
   </property>
  </action>
  <action name="actionPlots_to_Grid">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/grid_view.png</normaloff>:/icons/grid_view.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Plots to Grid</string>
   </property>
   <property name="toolTip">
    <string>Arrange plots to grid</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/connect.png</normaloff>:/icons/connect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to servers</string>
   </property>
  </action>
  <action name="actionSetup">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/preferences.png</normaloff>:/icons/preferences.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Setup</string>
   </property>
   <property name="toolTip">
    <string>Open setup dialog</string>
   </property>
  </action>
  <action name="actionChange">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Change Asset</string>
   </property>
   <property name="toolTip">
    <string>Change selected asset</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PlotManagerClass</class>
   <extends>QTreeWidget</extends>
   <header>plotmanagerclass.h</header>
   <slots>
    <slot>onAddPlot()</slot>
    <slot>onRemove()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd_Plot</sender>
   <signal>triggered()</signal>
   <receiver>plotsManager</receiver>
   <slot>onAddPlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionData_save</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onDataSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionData_load</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onDataLoad()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose_Plots</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onClosePlots()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd_Bus</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onAddBusAsset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPlots_to_Grid</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onGridPlots()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionConnect</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onConnect(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>tabCloseRequested(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onDataViewClose(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd_Asset</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onAddPipeAsset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove</sender>
   <signal>triggered()</signal>
   <receiver>plotsManager</receiver>
   <slot>onRemove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChange</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onChangeAsset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAddPipeAsset()</slot>
  <slot>onAbout()</slot>
  <slot>onDataSave()</slot>
  <slot>onDataLoad()</slot>
  <slot>onClosePlots()</slot>
  <slot>onAddBusAsset()</slot>
  <slot>onGridPlots()</slot>
  <slot>onConnect(bool)</slot>
  <slot>onChangeAsset()</slot>
  <slot>onDataViewClose(int)</slot>
 </slots>
</ui>
